src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_load_store_emitters.cpp

```diff
@@ -25,6 +25,89 @@ namespace ov::intel_cpu::aarch64 {
 using jit_generator = dnnl::impl::cpu::aarch64::jit_generator;
 using cpu_isa_t = dnnl::impl::cpu::aarch64::cpu_isa_t;
 
+// Helper function to load with large offset handling
+template <typename RegType>
+static void load_with_offset_check(jit_generator* h, const RegType& dst, const XReg& src, int offset) {
+    if constexpr (std::is_same_v<RegType, VReg> || std::is_same_v<RegType, QReg>) {
+        // Manual offset handling for VReg/QReg due to uni_ldr limitations
+        const int off_mod = offset % 16;
+        const int off_mul_vl = offset / 16;
+
+        if (off_mod == 0 && off_mul_vl >= 0 && off_mul_vl <= 4095) {
+            h->ldr(QReg(dst.getIdx()), ptr(src, static_cast<uint32_t>(offset)));
+        } else {
+            h->add_imm(h->X_DEFAULT_ADDR, src, offset, h->X_TMP_0);
```

Comment:
Just a question for better understanding: is it safe to use `h->X_DEFAULT_ADDR` and `h->X_TMP_0` here? How do we guarantee that `src` or `dst` are not `X_DEFAULT_ADDR` and `X_TMP_0`?

Reply:
YEs, we do. They are basically not used in code generation as inputs or outputs

Reply:
Is it possible to create a static vector with reserved registers somewhere? Just to avoid such questions in the future: we will have one place (vec initialization) where we will explain why they are in a blacklist. And we will be able to use them in the places we need using this vec. @a-sidorova @aobolensk what do you think?